### ISSUE TITLE
Metadata update- BC

### ISSUE THREAD

**Issue by ChanchalYa** [cancogen-virus-seq/portal#380]
Hi @edsu7,

Could you please update the `isolate name` for `BC_25661` to `hCoV-19/Canada/BC-BCCDC-25661/2021` in the portal?

Thanks,
Chanchal

**Comment by edsu7** [comment]
Hi @ChanchalYa,

Will do. I also noticed the `fasta_header_name` is matching the old one, would you like me to change that too?
```
"fasta_header_name": "hCoV-19/Canada/BC-BCCDC-25661/2020",
```

Cheers,
Edmund

**Comment by ChanchalYa** [comment]
Hi @edsu7 ,

Yes, please change the `fasta_header_name` as well.

Thank you!

**Comment by edsu7** [comment]
Hi @ChanchalYa,

`fasta_header_name` and `isolate name` have been updated for `BC_25661`.

Note because I had to alter the `fasta_header_name` in the file too, the sample had to be reuploaded with corrected header. ViralAI will have to account for this when auditing.

Old analysis was `1e07c5af-fa20-4982-87c5-affa20f9824e` suppressed.
New `8d36600e-62ad-4699-b660-0e62ade6996e`

![image](https://user-images.githubusercontent.com/22638361/232142177-c79d78dd-cc07-440d-97bc-0aacf58c6f95.png)

Cheers,
Edmund
